### ISSUE TITLE
Leetcode questions(unsolved) related to BinSearch

### ISSUE THREAD

**Issue by shruti170901** [shruti170901/Leetcode#4]
Solve one/more leetcode questions(unsolved in this repo) related to binSearch and send a PR

**Comment by anushkasinghal11** [comment]
Assign this to me. I'll do it in C++

**Comment by Siddhant-K-code** [comment]
Can I Do this ?

Also, We should differentiate/structure the things folder-wise String, Arrays, Binary Search, Heaps, etc.

**Comment by shruti170901** [comment]
Yes that can be differentiated but then there are also many questions that will fall in various categories. If you think you can organise these, go ahead!

**Comment by Siddhant-K-code** [comment]
pls check #16

**Comment by anushkasinghal11** [comment]
Hey!
I would like to work on this

**Comment by vedu2254** [comment]
I would like to contribute

**Comment by Dyx2222** [comment]
I would like to work on this.

**Comment by Navpreet1234** [comment]
I would like to work on this issue

**Comment by Manonmani19** [comment]
I would like to work on this.

**Comment by subhangi2731** [comment]
can you please assign this to me

**Comment by AnshikaGovil** [comment]
I would like to work on this
